streams/src/main/java/org/apache/kafka/streams/state/internals/CachingSessionStore.java

```diff
@@ -150,11 +151,14 @@ public void put(final Windowed<K> key, AGG value) {
 
     @Override
     public KeyValueIterator<Windowed<K>, AGG> fetch(final K key) {
+        Objects.requireNonNull(key, "key cannot be null");
         return findSessions(key, 0, Long.MAX_VALUE);
     }
 
     @Override
     public KeyValueIterator<Windowed<K>, AGG> fetch(K from, K to) {
+        Objects.requireNonNull(from, "<from> cannot be null");
```

Comment:
nit: remove the `<` and `>` here and next line. Just to keep it consistent with everywhere else we do this